ipywidgets_bokeh/src/manager.ts

```diff
@@ -191,17 +190,19 @@ export class WidgetManager extends HTMLManager {
   }
 
   override async _create_comm(target_name: string, model_id: string, data?: any, metadata?: any,
-      buffers?: ArrayBuffer[] | ArrayBufferView[]): Promise<IClassicComm> {
+      buffers?: ArrayBuffer[] | ArrayBufferView[]): Promise<shims.services.Comm> {
     const comm = (() => {
       const key = target_name + model_id
-      const comm = this._comms.get(key)
-      if (comm != null)
-        return comm
-      else {
-        const comm = this.kernel.createComm(target_name, model_id)
+      let comm = this._comms.get(key)
```

Comment:
linter still not happy:(
what about:
```suggestion
      let comm = this._comms.get(key)!
```

Reply:
This would lead to the linter complain on the next line about the `null` check, which is necessary.